feature-utils/silly-i18n/rollup-plugin.js

```diff
@@ -0,0 +1,57 @@
+import fs from "fs";
+import path from "path";
+import util from "util";
+import glob from "glob";
+
+const DIRNAME = "locales";
+const FILEEXT = ".json";
+const PLUGIN_NAME = "silly-i18n";
```

Comment:
nit: You're using `_NAME` here and `NAME` above.

Reply:
*Dirname* is an established term (in the context of programming) while *pluginname* is not, therefore I spelled it as two words. Anyway, I now renamed it to `PLUGIN`.